HoneyBadgers.ConsoleApp/UI/DataEditionMenu.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HoneyBadgers.Logic;
+
+namespace HoneyBadgers.ConsoleApp.UI
+{
+    class DataEditionMenu
+    {
+        private readonly IMovieRepository _movieRepository;
+        private readonly IUserRepository _usersRepository;
+
+        public DataEditionMenu(IUserRepository userRepository, IMovieRepository movieRepository)
+        {
+            _movieRepository = movieRepository;
+            _usersRepository = userRepository;
+        }
+
+        public void RunEditionMenu()
+        {
+            Console.Clear();
+            string prompt = "Which data would you like to edit?";
+            string[] options =
+            {
+                "Edit user data",
```

Comment:
myślę że tu mogłeś dodać opcje wyświetlenia movies i users, ale skoro i tak to zmieniamy na Web app to już nie ma potrzeby

Reply:
do przemyślenia na webówce. Tak czy siak printowanie jest połączone z edycją, można albo od razu przejść do edycji albo najpierw wyprintować i dopiero edytować.